Cargo.toml

```diff
@@ -17,14 +17,14 @@ members = [
 resolver = "2"
 
 [workspace.dependencies]
-zcash_address = { git = "https://github.com/zingolabs/librustzcash.git", tag = "calculate_tx_async" }
-zcash_client_backend = { git = "https://github.com/zingolabs/librustzcash.git", tag = "calculate_tx_async", features = ["lightwalletd-tonic", "orchard", "transparent-inputs"] }
-zcash_encoding = { git = "https://github.com/zingolabs/librustzcash.git", tag = "calculate_tx_async" }
-zcash_keys = { git = "https://github.com/zingolabs/librustzcash.git", tag = "calculate_tx_async", features = ["orchard"] }
+zcash_address = { git = "https://github.com/zingolabs/librustzcash.git", tag = "custom_notusk_to_taddr-1-g239c894b" }
+zcash_client_backend = { git = "https://github.com/zingolabs/librustzcash.git", tag = "custom_notusk_to_taddr-1-g239c894b", features = ["lightwalletd-tonic", "orchard", "transparent-inputs"] }
+zcash_encoding = { git = "https://github.com/zingolabs/librustzcash.git", tag = "custom_notusk_to_taddr-1-g239c894b" }
+zcash_keys = { git = "https://github.com/zingolabs/librustzcash.git", tag = "custom_notusk_to_taddr-1-g239c894b", features = ["orchard"] }
 zcash_note_encryption = "0.4"
-zcash_primitives = { git = "https://github.com/zingolabs/librustzcash.git", tag = "calculate_tx_async" }
-zcash_proofs = { git = "https://github.com/zingolabs/librustzcash.git", tag = "calculate_tx_async" }
-zcash_protocol = { git = "https://github.com/zingolabs/librustzcash.git", tag = "calculate_tx_async" }
+zcash_primitives = { git = "https://github.com/zingolabs/librustzcash.git", tag = "custom_notusk_to_taddr-1-g239c894b" }
+zcash_proofs = { git = "https://github.com/zingolabs/librustzcash.git", tag = "custom_notusk_to_taddr-1-g239c894b" }
+zcash_protocol = { git = "https://github.com/zingolabs/librustzcash.git", tag = "custom_notusk_to_taddr-1-g239c894b" }
```

Comment:
so, this change to librustzcash includes modifying calculate_proposed_transactions to literally substitute a custom function for key derivation in
its well-placed for convenience. its fair play in a librustzcash style

at this point im considering two other solutions that come to my mind for solving this problem, namely 
1) get  the right key inside zingolib
2) upgrade the key-unwrapping logic, fully inside lrz